src/TableView/TableView.jsx

```diff
@@ -78,15 +82,26 @@ export const getTableView = (schema, Table = TableComponent, isChildView = false
 
     componentDidMount() {
       if (!isChildView) {
-        const query = queryParse(this.props.location.search);
+        const {substringSearchEnabled, metadataSubstringSearchEnabled} = this.props;
+        const queryFormat = substringSearchEnabled ||
+          metadataSubstringSearchEnabled === false ?
```

Comment:
Please use consistent syntax when `substringSearchEnabled` and `metadataSubstringSearchEnabled` are used with logical operators. They are both boolean values so it's not necessary to compare them with boolean value.
```suggestion
          !metadataSubstringSearchEnabled ?
```